### ISSUE TITLE
Input `type="color"` - Screen readers unable to open via keyboard - (2036569945)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5578]
### Module: 
 16 Input &amp; Input Message 

### Violation: 
 Ensure custom controls are keyboard accessible 

![image](https://user-images.githubusercontent.com/5023024/202760108-f4e4ced4-c849-482f-a95e-fc7d7dd1e09b.png)

### WCAG Reference: 
- [2.1.1 Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html) 

### Instance ID: 
 2036569945 

### Severity: 
 8 

### Description: 
 [Issue]
The "color picker" control is not actionable with keyboard while a screen reader is running.

[User Impact]
Screen reader users will be prevented from accessing the functionality provided by these controls.

[Code Reference] 
 ```
<input aria-label="Color picker medium scale input" class="" enterkeyhint="" inputmode="" placeholder="" type="color">
 ``` 

### Note: 
 [Suggestion]
Ensure custom controls are keyboard accessible. Users must be able to navigate to and operate controls while a screen reader is running.

[Compliant Code Example] 
 ```
N/A
 ```
 

### Media Type: 
 Custom Controls 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
cc #5579

**Comment by geospatialem** [comment]
This is applicable in JAWS, but not NVDA. This also applies to the native `input type="color"`, which is being used on the component.

Closing as no code change can be made.
